docs/migration_2_to_3.rst

```diff
@@ -24,76 +25,82 @@ On this page you can read about points that changed corresponding to last stable
 Dispatcher
 ==========
 
-- :class:`Dispatcher` class no longer accepts the `Bot` instance into the initializer,
-  it should be passed to dispatcher only for starting polling or handling event from webhook.
-  Also this way adds possibility to use multiple bot instances at the same time ("multibot")
+- The :class:`Dispatcher` class no longer accepts a `Bot` instance in its initializer. 
+  Instead, the `Bot` instance should be passed to the dispatcher only for starting polling 
+  or handling events from webhooks. This approach also allows for the use of multiple bot 
+  instances simultaneously ("multibot").
 - :class:`Dispatcher` now can be extended with another Dispatcher-like
   thing named :class:`Router` (:ref:`Read more » <Nested routers>`).
-  With routes you can easily separate your code to multiple modules
-  and may be share this modules between projects.
+- With routes, you can easily modularize your code and potentially share these modules between projects.
 - Removed the **_handler** suffix from all event handler decorators and registering methods.
   (:ref:`Read more » <Event observers>`)
-- Executor entirely removed, now you can use Dispatcher directly to start polling or webhook.
-- Throttling method is completely removed, now you can use middlewares to control
-  the execution context and use any throttling mechanism you want.
+- The Executor has been entirely removed; you can now use the Dispatcher directly to start polling or handle webhooks.
```

Comment:
Can we convert the phrases to a single form?

```suggestion
- The Executor has been entirely removed; you can now use the Dispatcher directly to start polling or webhook handling.
```

or

```suggestion
- The Executor has been entirely removed; you can now use the Dispatcher directly to start poll updates or handle webhooks.
```

?

Reply:
This one is better👍